src/api/transfers/routes.js

```diff
@@ -61,7 +61,7 @@ module.exports = [{
         accept: Joi.string().optional().regex(regexAccept),
         'content-type': Joi.string().required().regex(regexContentType),
         'content-length': Joi.number().max(5242880),
-        date: Joi.date().format('ddd, D MMM YYYY H:mm:ss [GMT]').required(),
+        date: Joi.date().format('ddd, DD MMM YYYY HH:mm:ss [GMT]').required(),
```

Comment:
I'd suggest adding a separate schema for date, e.g.:
```
const utcDateSchema = Joi.date().format('ddd, DD MMM YYYY HH:mm:ss [GMT]').required();
```
and use it in all other places:
```
 headers: Joi.object({
    ...
    date: utcDateSchema,
```

Reply:
I did not see `Joi.date().format()` used anywhere else. I am not sure if this `date` header is important to us at all.

Reply:
...or you mean to not repeat it in this file, which I will fix now